### DISCUSSION TITLE
how to remove some support

### DISCUSSION THREAD

**Discussion by lungkao** [supermerill/SuperSlicer#2764] [General]
how to remove some support

![Screen Shot 2565-04-28 at 16 56 08](https://user-images.githubusercontent.com/2047778/165728178-13a6df03-05e2-45dd-9b30-3bb10277dc73.png)
![Screen Shot 2565-04-28 at 16 55 25](https://user-images.githubusercontent.com/2047778/165728198-54fe7bf9-71d8-4703-9418-e12cf98da2e7.png)

**Comment by supermerill** [comment]
not possible in 2.3
in 2.4, reduce the first layer expansion.

**Reply by lungkao** [reply]
Thank you
